src/cup1d/likelihood/lya_theory.py

```diff
@@ -12,12 +14,24 @@
 class Theory(object):
     """Translator between the likelihood object and the emulator. This object
     will map from a set of CAMB parameters directly to emulator calls, without
-    going through our Delta^2_\star parametrisation """
-
-    def __init__(self,zs,emulator=None,verbose=False,
-                    F_model_fid=None,T_model_fid=None,P_model_fid=None,
-                    include_metals=[],
-                    cosmo_fid=None,free_param_names=None):
+    going through our Delta^2_\star parametrisation"""
+
+    def __init__(
+        self,
+        zs,
+        emulator=None,
+        verbose=False,
+        F_model_fid=None,
+        T_model_fid=None,
+        P_model_fid=None,
+        z_star=3.0,
+        kp_kms=0.009,
+        include_metals=[],
+        cosmo_fid=None,
+        free_param_names=None,
+        sim_igm="mpg_central",
+        emu_type="nn",
```

Comment:
We should also clarify why you need this and also `emulator` above. I guess this is because you want to use this emu_type when no emulator is passed? In which case, maybe call this `default_emu_type`?

Reply:
I have deleted this flag, I managed to guess which type of emulator I am using dynamically